services/api-server/src/simcore_service_api_server/services/service_exception_handling.py

```diff
@@ -107,3 +109,14 @@ def backend_service_exception_handler(
         raise HTTPException(
             status_code=status_code, detail=detail, headers=headers
         ) from exc
+    except Exception as exc:
+        error_code = create_error_code(exc)
+        _logger.exception(
+            "Back-end service failed unexpectedly [%s]",
+            f"{error_code}",
+            extra={"error_code": error_code},
+        )
+        raise HTTPException(
+            status_code=status.HTTP_502_BAD_GATEWAY,
+            detail=MSG_INTERNAL_ERROR_USER_FRIENDLY_TEMPLATE.format(error_code),
+        ) from exc
```

Comment:
I would not place this here. There is a general exception handler which is supposed to handle any "rouge" Exception. In my opinion this should also not be converted into a 502 because the exception could be due to some internal issue in the api-server. In my mind the 502 is returned in the following two cases:
1. An *unexpected* status code is returned from the backend. This should mean a status code which is not specified in the openapi specification of the backend service. 
2. A pydantic validation error coming from deserializing the body of the response from the backend. 

These are the two `except`s above.

Reply:
we need to check this again offline so we sync ...